app/services/atmosphere/billing_service.rb

```diff
@@ -72,48 +92,46 @@ def self.bill_appliance(appl, billing_time, message, apply_prepayment = true)
       end
     end
 
-    # This method calculates the amount which should be billed to a specific appliance on billing_time.
+    # This method calculates the amount which should be billed to a specific deployment on billing_time.
     # It does not actually incur the charge, merely returns the appropriate information.
     # billing_time specifies when the billing action is actually performed
     # apply_prepayment determines whether the method should include prepayment period (of length specified in @appliance_prepayment_interval)
-    # - This is usually true, but will be false when calculating final charge for an appliance which is being shut down.
-    def self.calculate_charge_for_appliance(appl, billing_time, apply_prepayment = true)
-      # By default, the billing process should extend the validity of the appliance until now+1.hour.
-      billable_time = self.calculate_billable_time(appl, billing_time, apply_prepayment)
+    # - This is usually true, but will be false when calculating final charge for a deployment which is being shut down.
```

Comment:
This also can be done later: when documentation for the method is created can you use [Rdoc format](http://rdoc.sourceforge.net/doc/index.html)?